.github/workflows/dependency-check.yml

```diff
@@ -1,12 +1,23 @@
-name: Go Dependency Checker
+name: Dependency Vulnerability Check
 
 on: [push]
 jobs:
-  build:
+  Go:
     runs-on: ubuntu-latest
     steps:
-    - uses: actions/checkout@v1
-    - name: Check
-      uses: sonatype-nexus-community/nancy-github-action@76cc24d43c279e8f357d4dccf5ce226835f8385f
+    - name: Check out code
+      uses: actions/checkout@v2
+
+    - name: Set up Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.15
+      id: go
+
+    - name: Write Go Modules list
+      run: go list -json -m all > go.list
```

Comment:
Is this `go.list` something that nancy uses?!

Reply:
Yes, the normal command to run it locally with Go modules is `go list -json -m all | nancy sleuth`. Nancy also supports `dep` so there seem to be a couple of ways of coercing dependency managers to fit this standardized file, then the Nancy Github Action assumes that file is present.